crates/sui-indexer/src/handlers/committer.rs

```diff
@@ -103,9 +114,11 @@ pub async fn start_tx_checkpoint_commit_task<S>(
         // this is a one-way flip in case indexer falls behind again, so that the objects snapshot
         // table will not be populated by both committer and async snapshot processor at the same time.
         if latest_committed_cp + OBJECTS_SNAPSHOT_MAX_CHECKPOINT_LAG > latest_fn_cp {
+            info!("Flipping object_snapshot_backfill_mode to false because objects_snapshot is close to up-to-date.");
```

Comment:
@wlmyng the != is the initial check to see if we should start with backfill mode at all, if so like backfill from genesis, it's here when the backfill is close to be done, we flip the backfill mode to false

Reply:
Oh I see, my eyes were attributing this to the other file instead

Reply:
Could we add a comment for the check 🙏